pkg/plugins/runtime/k8s/webhooks/validation.go

```diff
@@ -85,7 +85,7 @@ func (h *validatingHandler) Handle(_ context.Context, req admission.Request) adm
 		if err := core_model.Validate(coreRes); err != nil {
 			if kumaErr, ok := err.(*validators.ValidationError); ok {
 				// we assume that coreRes.Validate() returns validation errors of the spec
-				return convertSpecValidationError(kumaErr, coreRes.Descriptor().IsTargetRefBased, k8sObj)
+				return convertSpecValidationError(kumaErr, coreRes.Descriptor().IsPluginOriginated, k8sObj)
```

Comment:
so I'm assuming `IsPluginOriginated = IsTargetRefBased + core plugin resources`?

Reply:
in particular both IsPluginOriginated => has `spec` and IsTargetRefBased => has `spec`